native/arguments.go

```diff
@@ -49,6 +50,23 @@ func (b BaselineArguments) Configure(_ []string) ([]string, string, error) {
 	return newArguments, "", nil
 }
 
+// BootArguments provides a set of arguments that are provided by Spring Boot via META-INF/native-image/argfile
```

Comment:
This could be named `NativeArguments` or something similar, with all other references to `Boot` and `Spring Boot` replaced replaced with terminology that's more generic.